Makefile

```diff
@@ -277,3 +277,9 @@ helm-update-snapshots: _helm_setup ## Update helm unittest snapshots
 
 helm-update-snapshots-no-deps: ## Update helm unittest snapshots without rebuilding dependencies
 	$(MAKE) -C $(HELM_CHART_DIR) update-snapshots
+
+##@ Chainsaw
+
+.PHONY: chainsaw-tests
+chainsaw-tests: ## Run chainsaw tests
+	chainsaw test ./tests/chainsaw
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +281 to +286]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Nit: imo Chainsaw is the implementation of end-to-end tests, so perhaps we call this target `e2e-tests` in-case we change the implementation later?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/ngrok/ngrok-operator/506?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
Done